test/framework/resource/k8s/deployment/manager.go

```diff
@@ -40,6 +41,39 @@ type defaultManager struct {
 	k8sClient client.Client
 }
 
+func (m *defaultManager) ScaleDeployment(ctx context.Context, name string, namespace string, replicas int) error {
+	var deployment *appsv1.Deployment
```

Comment:
will this ever work? since deployment == nil

Reply:
looks like this is unused code, good catch!